Description: Skip non-dated log files
 ntpd creates both dated (e.g. peerstats.YYMMDDD) and non-dated
 (e.g. peerstats) log files.  ntpviz should only look at the dated
 files to avoid loading duplicate data.
Forwarded: https://gitlab.com/NTPsec/ntpsec/merge_requests/226
Origin: vendor
Author: Richard Laager <rlaager@wiktel.com>
Last-Update: 2016-12-17
Applied-Upstream: 0.9.7, https://gitlab.com/NTPsec/ntpsec/commit/8d696585fb3dc092c472a390da006f74ea1af12e
--- a/pylib/statfiles.py
+++ b/pylib/statfiles.py
@@ -83,7 +83,10 @@
                  "temps", "gpsd"):
             lines = []
             try:
-                for logpart in glob.glob(os.path.join(statsdir, stem) + "*"):
+                pattern = os.path.join(statsdir, stem)
+                if stem != "temps" and stem != "gpsd":
+                    pattern += "."
+                for logpart in glob.glob(pattern + "*"):
                     # skip files older than starttime
                     if starttime > os.path.getmtime(logpart):
                         continue
